examples/configs/privateai/pii_detection/config.yml

```diff
@@ -0,0 +1,27 @@
+models:
+  - type: main
+    engine: openai
+    model: gpt-3.5-turbo-instruct
+
+rails:
+  config:
+    privateai:
+      server_endpoint: https://api.private-ai.com/cloud/v3/process/text
+      api_key: <YOUR_API_KEY>  # get your API key from https://portal.private-ai.com
```

Comment:
Can we remove this? Either defining an environment variable name that is fixed and the user can set. Or allow the user to pass the environment variable name, but I'd suggest the former.